kyuubi-common/src/main/scala/org/apache/kyuubi/operation/OperationManager.scala

```diff
@@ -36,7 +38,7 @@ import org.apache.kyuubi.shaded.hive.service.rpc.thrift._
  */
 abstract class OperationManager(name: String) extends AbstractService(name) {
 
-  final private val handleToOperation = new java.util.HashMap[OperationHandle, Operation]()
+  final private val handleToOperation = new ConcurrentHashMap[OperationHandle, Operation]()
```

Comment:
The method using `handleToOperation` has synchronized block, do we still need this change?

https://github.com/apache/kyuubi/blob/0165d5e3a26cc8353a9f7135e8fa4a9c4ae99e78/kyuubi-common/src/main/scala/org/apache/kyuubi/operation/OperationManager.scala#L98-L101

Reply:
thanks, reverted